sdk/resourcemanager/Microsoft.Azure.ResourceManager.Moq/README.md

```diff
@@ -0,0 +1,47 @@
+# Newtonsoft.Json support for Azure Core shared client library for .NET
+
+The [Azure.Core package][azure_core_package] contains types shared by the latest Azure SDK client libraries. This `Newtonsoft.Json` compatibility library:
```

Comment:
Update the content in this file for the Moq extension library.

Reply:
I will do that later when I finish the code